src/chocolatey/infrastructure.app/configuration/EnvironmentSettings.cs

```diff
@@ -134,10 +135,10 @@ public static void SetEnvironmentVariables(ChocolateyConfiguration config)
 
         private static void SetLicensedEnvironment(ChocolateyConfiguration config)
         {
-            if (!config.Information.IsLicensedVersion) return;
-
             Environment.SetEnvironmentVariable("ChocolateyLicenseType", config.Information.LicenseType);
 
+            if (!(config.Information.IsLicensedVersion && config.Information.IsLicensedAssemblyLoaded)) return;

```

Comment:
Let us use curly braces for any new code

```suggestion
            if (!(config.Information.IsLicensedVersion && config.Information.IsLicensedAssemblyLoaded))
            {
                return;
            }

```

Also, any reason you changed this to happen after setting the environment variable?
This will most likely break some of our test kitchen tests (if memory serves me right).

Reply:
I moved it to after setting that particular environment variable because the `chocolateyInstall.ps1` script for the extension explicitly checks it and fails if it's not set.

As for reformatting the if statement, I left it without braces as that's the way I found it. But my preference is to have them as well, so I'll get that updated.